src/figvalidationservice/src/test/java/ca/bc/gov/open/pssg/rsbc/dps/figvalidationservice/ValidateApplicantForSharingControllerTest.java

```diff
@@ -0,0 +1,76 @@
+package ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice;
+
+import ca.bc.gov.open.ords.figcr.client.api.model.ValidateApplicantForSharingOrdsResponse;
+import ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice.exception.FigaroValidationServiceException;
+import ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice.types.ValidateApplicantForSharingResponse;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class ValidateApplicantForSharingControllerTest {
+
+
+    public static final String VALID_STATUS_CODE = "0";
```

Comment:
I would suggest making these variables private unless you are using them in another class.

Reply:
Yes thanks

Reply:
changed